thunder/torch/__init__.py

```diff
@@ -5119,13 +5119,15 @@ def _interpolate_scale_factor_helper(
         )
 
     # perform nearest up/down-sampling
-    def nearest_sampler(t, input_dim, output_dim, *, scale, dim):
+    def nearest_sampler(t, input_dim, output_dim, *, scale, dim, exact):
```

Comment:
Would you add type annotations to this function? I think they would greatly improve its readability